barcalendar.py

```diff
@@ -22,9 +22,87 @@
 import re
 import time
 
+import atexit
+import logging
```

Comment:
[nit] These two lines from the stdlib, so should be alphabetized into the first block of imports.

Reply:
Done.